src/web/src/components/Banner.tsx

```diff
@@ -110,18 +110,14 @@ type BannerProps = {
   onVisibilityChange: (visible: boolean) => void;
 };
 
-const getQuote = () => {
-  return quotes[Math.floor(Math.random() * quotes.length)];
-};
-
 export default function Banner({ onVisibilityChange }: BannerProps) {
   const classes = useStyles();
   const [gitInfo, setGitInfo] = useState({
     gitHubUrl: '',
     sha: '',
     version: '',
   });
-  const studentQuote = getQuote();
+  const [studentQuote] = useState(() => quotes[Math.floor(Math.random() * quotes.length)]);
```

Comment:
Let's consider doing what @DukeManh suggested and move this out of the component.  Put it on line 113 above and do:

```ts
const studentQuote = quotes[Math.floor(Math.random() * quotes.length)]);
```

Then you can use it in the component, but it shouldn't change its value on re-render.

Reply:
Perfect. I tried the solution but didn't quite understand we had to move it out, and it had not worked. This works now! Thank you.